py_gtfs_rt_ingestion/ingest.py

```diff
@@ -179,8 +173,7 @@ def lambda_handler(event: LambdaDict, context: LambdaContext) -> None:
     logging.info("Context:%s", context)
 
     try:
-        files = event["files"]
-        main(files)
```

Comment:
just pass the event over.

i still think its a good idea to have the handler separate from main.